### PR TITLE
feaure/102/database-connection

### PR DESCRIPTION
**Pull request by TimMo-dev**
## PR purpose description
To setup typeORM. 

## Quality check-list
-[ ] I have performed a self-review
-[ ] Unit tests have been created for the code 
-[ ] I have used linting/static code check

### CODE REVIEW

**Comment by TimMo-dev** [comment]
Maybe i should remove package-lock.json xd

File: programming-learning-tool/server/data-source.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import "reflect-metadata"
+import  { DataSource }from "typeorm"
+import { ProgrammingTask } from "./models/ProgrammingTask"
+export const AppDataSource = new DataSource({
+    type: "postgres",
+    host: "localhost",
+    port: 5431,
+    username: "user",
+    password: "password",
```
Comment (JakobTopholt) [inline programming-learning-tool/server/data-source.ts:9]: ?

Comment (TimMo-dev) [reply programming-learning-tool/server/data-source.ts:11]: det er bare en test container, dog er der et argument for at det skulle se anderledes ud på en production database. F.eks. brug af en env fil

**Review by mpmunch — APPROVED** [review]
I could seed the database and the tasks appeared at the front page - LGTM

**Pull request merged**